apps/i18n/common/en_us.json

```diff
@@ -1986,6 +1986,13 @@
   "plugged": "Plugged",
   "pluggedLessonsNote": "*Online or ‘plugged’ lessons are automatically marked as complete on your behalf once 80% of your class has completed 60% of the available lesson.",
   "picturePassword": "Picture password",
+  "policyCompliance_parentalPermissionModal_newRequestForm_title": "Heads up! Your account may soon be locked.",
+  "policyCompliance_parentalPermissionModal_newRequestForm_text1": "Starting on {lockoutDate}, **you will need your parent or guardian’s permission to keep your personal login** with Code.org.",
+  "policyCompliance_parentalPermissionModal_newRequestForm_text2": "To avoid having your account locked, please share your parent or guardian’s email address so they can grant you permission.",
+  "policyCompliance_parentalPermissionModal_updateRequestForm_title": "Thanks! We’ve contacted your parent or guardian.",
+  "policyCompliance_parentalPermissionModal_updateRequestForm_text1": "Didn’t receive anything? Update your parent or guardian’s email below or send another request. Note, your account will be locked if we do not receive your parent/guardian’s permission by {lockoutDate}.",
+  "policyCompliance_parentalPermissionModal_updateRequestForm_text2": "You can also provide this email address at any time from [your profile]({profileUrl}).",
+  "policyCompliance_parentalPermissionModal_lastEmailSentAt": "Last email sent at {sendingTime}",
```

Comment:
I've already created some strings under the namespace `childAccountPolicy_`.
Would it be reasonable to stick to that namespace? If you feel strongly about using `policyCompliance_` I'm okey switching the strings I've added, but I think we should stay consistent.
https://github.com/code-dot-org/code-dot-org/blob/9cb8ea19ea80d65a38a253c8c0b7bcebc4f48493/apps/i18n/common/en_us.json#L412-L413

Reply:
I think if your `ChildAccountPolicy` component were in the `policy_compliance` path, a good name for the string key would then be `policyCompliance_childAccountPolicy_learnMore`, etc. I generally didn't use an `_` when I would do this... that seems nicer though.

That said, I don't think I care too much that the actual name of the component is there... just the major grouping. These strings end up moving around as things are updated (or reused elsewhere) that they shouldn't be too tied to the component name, in my opinion, and be more focused on their relationship to their function. So, for the `LockoutPanel`, which I put within the `sessions` grouping because at the time it was supported by the sessions controller, I think I made strings like `sessionLockoutNewAccountHeader`, etc. So, I expect this to be used by stuff within the `sessions` path, so `session`... they have to do with the "`Lockout`" function... and then the general key.

If I was redesigning everything from scratch, I'd probably want to define the strings in the components themselves and then pull them out instead of trying to define every string in one file. Haha what can you do though.